lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -27,6 +27,78 @@ class WP_Font_Library {
 		'woff2' => 'font/woff2',
 	);
 
+	/**
+	 * Font collections.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	public static $collections = array();
+
+	/**
+	 * Register filter to extend the library with font collections.
+	 *
+	 * @since 6.4.0
+	 */
+	public static function register_filters() {
+		add_filter(
+			'register_font_collection',
```

Comment:
Prefix the filter's name with `wp_` to avoid naming collisions in the wild.

Reply:
Done!